CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added a setting to control which CrossServer RPC implementation is used. Both feature mentioned below are only enabled when the RoutingWorker is the chosen implementation. Spatial commands are still the default for now.
 - Added reliable CrossServer RPC. Reliable CrossServer RPC now require a sender actor which will be the reference point for ordering in a multi-worker environment. An additional UFUNCTION Tag, Unordered, was added to opt-out of this requirement.
 - Added NetWriteFence UFUNCTION Tag. This tag is used when Network writes to an actor should be ordered with regard to updates to another actor. This is relevant in worker recovery/snapshot reloading to get some ordering guarantees when SpatialOS can write updates to entities in any order.
+- Added support for clients to disconnect during a test in the automated test framework.
```

Comment:
Maybe internal section for now? While we don't fully support SpatialFunctionalTest externally

Reply:
but maybe it's ok to leave it here

Reply:
+1 to taking this internal, not to hide it from customers (we're open source) but to direct the eyes of internal users at it.

Reply:
Moved to internal section.